src/apps/frontend/components/form-control/index.tsx

```diff
@@ -10,8 +10,8 @@ const FormControl: React.FC<PropsWithChildren<FormControlProps>> = ({
   error,
   label,
 }) => (
-  <>
-    <label className="mb-2.5 block font-medium text-black dark:text-white">
+  <div>
```

Comment:
I see we are probably trying to stack the form controls vertically, with mb on each component. Can we use a layout so each component does not have mb?